packages/adblocker/src/engine/bucket/cosmetic.ts

```diff
@@ -347,7 +347,6 @@ export default class CosmeticFilterBucket {
     getRulesFromDOM = true,
     getRulesFromHostname = true,
 
-    hidingStyle = DEFAULT_HIDING_STYLE,
```

Comment:
I assume, the type below can be removed as well:

```
hidingStyle?: string | undefined;
```

Reply:
Good catch!